governance/board-sync.py

```
@@ -0,0 +1,354 @@
+            if content.get("__typename") != "Issue" or not content.get("repository"):
+                continue  # 非 issue 条目（草稿/PR）不参与对账
+            vals = {}
+            for fv in (it.get("fieldValues") or {}).get("nodes") or []:
+                fname = ((fv.get("field") or {}).get("name"))
+                if not fname:
+                    continue
+                if fv["__typename"] == "ProjectV2ItemFieldSingleSelectValue":
+                    vals[fname] = fv.get("name", "")
+                elif fv["__typename"] in ("ProjectV2ItemFieldTextValue", "ProjectV2ItemFieldNumberValue"):
+                    vals[fname] = fv.get("text", fv.get("number"))
+            closed_labels = [l["name"] for l in (content.get("labels") or {}).get("nodes") or []]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)